analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/RequestsWithExcessiveLength.vb

```diff
@@ -13,13 +13,13 @@ Namespace Tests.TestCases
         End Function
 
         <HttpPost>
-        <RequestSizeLimit(2_000_001)> ' Noncompliant ^10#27 {{Make sure the content length limit is safe here.}}
+        <RequestSizeLimit(8_000_001)> ' Noncompliant ^10#27 {{Make sure the content length limit is safe here.}}
```

Comment:
what does `^10#27` mean?

also, the comment in line 10 confuses me ... `^10#25`?

(same question for analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/RequestsWithExcessiveLength_CustomValues.vb)

Reply:
^10#27 means the issue begins on column 10 and has a length of 27

Reply:
could you please use the alternative syntax `//        ^^^^^^^^^^^^` because this is a visual way of showing what's underlined. Otherwise, we cannot really verify in github, and it's painful to verify in the IDE, as well.

Reply:
For CS it is with the `// ^^^^^^^` syntax but for VB it is not possible. It is nor permitted to insert a comment between the attribute and the method.

Reply:
maybe you can use `'         ^^^^^@-1` for the VB case? see [ex1](https://github.com/SonarSource/sonar-dotnet/blob/d9451117ff541bf5aa2e83998ba7b945e8f78b5a/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/BooleanLiteralUnnecessary.vb#L46) , [ex2](https://github.com/SonarSource/sonar-dotnet/blob/d9451117ff541bf5aa2e83998ba7b945e8f78b5a/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CognitiveComplexity.vb#L359) , [ex3](https://github.com/SonarSource/sonar-dotnet/blob/d9451117ff541bf5aa2e83998ba7b945e8f78b5a/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CognitiveComplexity.vb#L482)